doc/classes/ProjectSettings.xml

```diff
@@ -1568,7 +1568,9 @@
 		<member name="rendering/2d/snap/snap_2d_vertices_to_pixel" type="bool" setter="" getter="" default="false">
 		</member>
 		<member name="rendering/anti_aliasing/quality/msaa" type="int" setter="" getter="" default="0">
-			Sets the number of MSAA samples to use (as a power of two). MSAA is used to reduce aliasing around the edges of polygons. A higher MSAA value results in smoother edges but can be significantly slower on some hardware. See also bilinear scaling 3d [member rendering/scaling_3d/mode] for supersampling, which provides higher quality but is much more expensive.
+			Sets the number of MSAA samples to use (as a power of two). MSAA is used to reduce aliasing around the edges of polygons. MSAA can also smooth out the edges of alpha-tested materials if alpha antialiasing is enabled on the material (see [member BaseMaterial3D.alpha_antialiasing_mode]). A higher MSAA value results in smoother edges but can be significantly slower on some hardware. See also [member rendering/scaling_3d/mode] and [member rendering/scaling_3d/scale] for supersampling, which provides higher quality but is much more expensive.
+			[b]Note:[/b] MSAA does not operate on shader-induced aliasing, which means it cannot smooth out specular aliasing. To smooth out specular aliasing, a screen-space antialiasing method must be used (see [member rendering/anti_aliasing/quality/screen_space_aa]).
+			[b]Note:[/b] If [member rendering/anti_aliasing/quality/msaa] is set to a value other than [b]Disabled[/b], the depth prepass ([member rendering/driver/depth_prepass/enable]) becomes considerably more expensive to perform. At high MSAA values, you may find that disabling depth prepass may improve average performance, even in more complex scenes. However, in very complex scenes, disabling the depth prepass will still decrease performance, regardless of MSAA level.
```

Comment:
I'm not sure this line is accurate, how MSAA/ the depth prepass impact performance is a complex relationship. I would keep this line a little more vague and avoid giving absolute performance advice. Instead say something like" the depth prepass helps improve performance in most cases by eliminating pixel overdraw, but especially when MSAA is used, can be quite expensive. You can try different MSAA/Depth prepass settings to find a configuration that is right for your project.